src/ReactiveUI.Tests/ActivationTests.cs

```diff
@@ -45,12 +42,14 @@ public DerivedActivatingViewModel()
     public class ActivatingView : ReactiveObject, IViewFor<ActivatingViewModel>
     {
         ActivatingViewModel viewModel;
-        public ActivatingViewModel ViewModel {
+        public ActivatingViewModel ViewModel
```

Comment:
I think this matches the Rebracer settings (`NewLines_Braces_Property`) and would be consistent with ReactiveCommand line 572 for example.